pkg/eventcache/eventcache.go

```diff
@@ -19,14 +19,11 @@ import (
 	"github.com/cilium/tetragon/pkg/server"
 )
 
-// garbage collection states
 const (
-	threeStrikes = 3
-)
-
-// garbage collection run interval
-const (
-	eventRetryTimer = time.Second * 10
+	// garbage collection retries
+	cacheStrikes = 15
+	// garbage collection run interval
+	eventRetryTimer = time.Second * 2
```

Comment:
The 10seconds was about balancing GC performance cost vs getting events out. We should add a configMap option for this I think.

Reply:
Created https://github.com/cilium/tetragon/issues/296 to keep track of that.